IGNITE-20882 Append performance tests

nva opened the pull request:
Thank you for submitting the pull request.

To streamline the review process of the patch and ensure better code quality
we ask both an author and a reviewer to verify the following:

### The Review Checklist
- [ ] **Formal criteria:** TC status, codestyle, mandatory documentation. Also make sure to complete the following:  
\- There is a single JIRA ticket related to the pull request.  
\- The web-link to the pull request is attached to the JIRA ticket.  
\- The JIRA ticket has the Patch Available state.  
\- The description of the JIRA ticket explains WHAT was made, WHY and HOW.  
\- The pull request title is treated as the final commit message. The following pattern must be used: IGNITE-XXXX Change summary where XXXX - number of JIRA issue.
- [ ] **Design:** new code conforms with the design principles of the components it is added to.
- [ ] **Patch quality:** patch cannot be split into smaller pieces, its size must be reasonable.
- [ ] **Code quality:** code is clean and readable, necessary developer documentation is added if needed.
- [ ] **Tests code quality:** test set covers positive/negative scenarios, happy/edge cases. Tests are effective in terms of execution time and resources.

### Notes
- [Apache Ignite Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide)

AMashenkov commented on modules/table/src/integrationTest/java/org/apache/ignite/internal/benchmark/CriteriaEmbeddedNodeBenchmark.java:
```diff
@@ -0,0 +1,154 @@
+        }
+
+        @Nullable Tuple get(Tuple key) {
+            return CLUSTER_NODES.get(0).tables().table(TABLE_NAME).recordView().get(null, key);
```
`CLUSTER_NODES.get(0).tables().table(TABLE_NAME).recordView()` this may costs as much as `get()` operation.

nva commented:
> Why you decided to duplicate existing infrastructure from [ignite-runner](https://issues.apache.org/jira/browse/IGNITE-runner)?

It is not possible to reuse `integrationTest` from other module. I removed duplication and moved benchmarks to the ignite-runner module like it was done for sql.
